[rospy.client][INFO] 2021-09-05 17:50:11,951: init_node, name[/server_endpoint], pid[30163]
[xmlrpc][INFO] 2021-09-05 17:50:11,951: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-09-05 17:50:11,951: Started XML-RPC server [http://wng:35301/]
[rospy.impl.masterslave][INFO] 2021-09-05 17:50:11,951: _ready: http://wng:35301/
[rospy.registration][INFO] 2021-09-05 17:50:11,952: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-09-05 17:50:11,952: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-09-05 17:50:11,953: ROS Slave URI: [http://wng:35301/]
[rospy.init][INFO] 2021-09-05 17:50:11,954: registered with master
[rospy.rosout][INFO] 2021-09-05 17:50:11,954: initializing /rosout core topic
[rospy.rosout][INFO] 2021-09-05 17:50:11,955: connected to core topic /rosout
[rospy.simtime][INFO] 2021-09-05 17:50:11,955: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-09-05 17:50:11,965: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-09-05 17:50:12,192: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-09-05 17:50:20,728: Connection from 192.168.1.146
[rosout][INFO] 2021-09-05 17:50:20,732: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-09-05 17:50:20,830: topic[/execution_check] adding connection to [http://wng:35301/], count 0
[rospy.internal][INFO] 2021-09-05 17:50:20,831: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-09-05 17:51:09,078: signal_shutdown [signal-2]
[rospy.internal][ERROR] 2021-09-05 17:51:09,082: Unable to initiate TCP/IP socket to wng:44189 (http://wng:35301/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2021-09-05 17:51:09,082: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-09-05 17:51:09,082: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-09-05 17:51:09,082: topic[/execution_check] removing connection to http://wng:35301/
[rospy.impl.masterslave][INFO] 2021-09-05 17:51:09,083: signal-2
[rospy.core][INFO] 2021-09-05 17:51:09,214: signal_shutdown [atexit]
